examples/plot_04_tracking_items.py

```diff
@@ -0,0 +1,160 @@
+"""
+.. _example_historization_items:
+
+==================================
+Tracking items using their history
+==================================
+
+This example illustrates how skore can be used to track some items using their history,
+for example tracking some ML metrics over time.
+"""
+
+# %%
+import subprocess
+import time
+
+import matplotlib.image as mpimg
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import plotly.express as px
+from sklearn import datasets
+from sklearn.linear_model import Lasso
+
+import skore
```

Comment:
Please can you move the imports to the cells that use them, as in other examples?

Reply:
Done